.github/workflows/ci.yml

```diff
@@ -140,6 +140,32 @@ jobs:
           .
         ./docker/coffea_rados_parquet/run.sh
 
+  testservicexprocessor:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8]
+    name: test serviceX Processor (${{ matrix.os }}) - python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    needs: [test]
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install -q -e .[dev,servicex]
+        python -m pip list
+    - name: Test with pytest
+      env:
+        ARROW_PRE_0_15_IPC_FORMAT: 1
```

Comment:
I've forgotten what this magic env var does, @lgray ?

Reply:
That's for spark, since it uses the arrow ipc format from before arrow v0.15